lms/djangoapps/instructor_task/tests/test_tasks_helper.py

```diff
@@ -43,10 +43,29 @@
     generate_students_certificates,
 )
 from openedx.core.djangoapps.util.testing import ContentGroupTestCase, TestConditionalContent
+from teams.tests.factories import CourseTeamFactory, CourseTeamMembershipFactory
+
+
+class TestInstructorGradeReportBase(TestReportMixin, InstructorTaskCourseTestCase):
```

Comment:
Nit: I think it is useful to name base classes so that they don't look like they have tests. I'd call this ``InstructorGradeReportTestCase``.